### ISSUE TITLE
Disable Rotation of camera

### ISSUE THREAD

**Issue by davidlondono** [remirobert/CameraEngine#18]
my app is just on Portrait, but when I turn the divice, the camera turns, but the rest of the layout dont, so it appears rotated

**Comment by remirobert** [comment]
Let me check that thanks for reporting this issue.

**Comment by remirobert** [comment]
fix in this commit: 0af6dc357fa70aa3466914f867d6da0db1209bee

I had a bolean variable, where you can set it to **disable** or **enable** the rotation camera. The default value is false.
```swift
self.cameraEngine.rotation = true
self.cameraEngine.rotation = false
```

**Comment by davidlondono** [comment]
thanks
